RecoJets/JetProducers/plugins/FixedGridRhoProducerFastjetFromRecHit.cc

```diff
@@ -55,6 +58,11 @@ class FixedGridRhoProducerFastjetFromRecHit : public edm::stream::EDProducer<> {
 
   const edm::ESGetToken<EcalPFRecHitThresholds, EcalPFRecHitThresholdsRcd> ecalPFRecHitThresholdsToken_;
   const edm::ESGetToken<CaloGeometry, CaloGeometryRecord> caloGeometryToken_;
+
+  // following are needed to grab HCal thresholds from GT
+  edm::ESGetToken<HcalPFCuts, HcalPFCutsRcd> hcalCutsToken_;
+  bool cutsFromDB;
+  HcalPFCuts const *paramPF = nullptr;
```

Comment:
This is a minor comment, and probably also a late one (sorry for that), but you could follow the prescriptions here and use trailing underscore for all the newly addedd class members, as it is done for all other ones in this class, and also make cutsFromDB const, e.g.
```suggestion
  const bool cutsFromDB_;
  HcalPFCuts const *paramPF_ = nullptr;
```

Reply:
sure, no worries. 
just waiting to see if test results are meaningful, then I will address these comments